src/index.js

```diff
@@ -116,14 +115,14 @@ class DelegatedPeerRouting {
             // Track the addresses, so we can yield them when done
             result.responses.forEach(response => {
               peers.set(response.id, {
-                id: PeerId.createFromCID(response.id),
+                id: PeerId.createFromB58String(response.id),
```

Comment:
Do we want to make this change? I remember we wanted to move towards not using b58 peer IDs and this will mean changing this back again

Reply:
`PeerId.createFromCID(response.id)` here was accidentally taking a base58 encoded string so can't be used any more.  Annoyingly the DHT result type 2 has `reponse.id` as a `CID` instance whereas type 1 has it as a string.

I've changed it to use `PeerId.parse` so we can add support for extra bases there without having to refactor everything in future.